dl/src/main/python/nn/layer.py

```diff
@@ -337,6 +392,14 @@ def __init__(self,
                                    num_input_dims)
 
 
+def build_model():
```

Comment:
This is a utility function for loadcaffe doctest, as we already have separate loadcaffe test, I remove it.